docs/auto-tag-orders-with-utm-parameters/script.liquid

```diff
@@ -112,22 +126,29 @@
 {% elsif event.topic == "mechanic/shopify/bulk_operation" %}
   {% if event.preview %}
     {% capture bulkOperation_objects_json %}
```

Comment:
One more very tiny change, which wasn't in-scope for this work but I'm requesting anyway since we're here: use `_jsonl` as the variable suffix here (instead of `_json`), updating the variable reference a couple lines down.